mappings/net/minecraft/client/particle/BlockLeakParticle.mapping

```diff
@@ -14,8 +27,8 @@ CLASS net/minecraft/unmapped/C_huhweqoj net/minecraft/client/particle/BlockLeakP
 		FIELD f_kwdmuich spriteProvider Lnet/minecraft/unmapped/C_eagntdua;
 		METHOD <init> (Lnet/minecraft/unmapped/C_eagntdua;)V
 			ARG 1 spriteProvider
-	CLASS C_cfnbjhqg DripstoneLavaDrip
-	CLASS C_dtvvaenu FallingDripstoneWaterFactory
+	CLASS C_cfnbjhqg FallingDripstoneFluid
```

Comment:
Would `FallingDripstoneParticle` make more sense here?

Reply:
`FallingDripstoneFluid` fits the naming scheme of the other classes, `FallingDripstoneParticle` is different. None of them mention particle.
other examples:
- Dripping
- DrippingLava
- old name, DripstoneLavaDrip

Reply:
I suppose that makes sense, well I'm not a fan of any of them personally, it makes sense and is consistent.